packages/ckeditor5-language/README.md

```diff
@@ -0,0 +1,16 @@
+CKEditor 5 language feature
```

Comment:
Let's be consistent here – "Language feature".

Also, cc @AnnaTomanek, do you think it makes sense to call it like this?

Reply:
"Language" looks fine (as it's consistent with CKE4). Another option would be "text language" as it then would be clear that it's not about the UI language. In WCAG the success criterion is named "Language of Parts", the `lang` attribute itself defines the language of text/element. To make it short, we can go with "language" as the feature name and then use other keywords (language of text, language of content, language of text parts) in the docs.

Reply:
I'm for "text language" and renaming this plugin. I'd prefer to avoid confusion as much as we can.

Reply:
`text language` would also improve options config. For now, I had to go with `languageList` to avoid duplication with `language`. Not sure if you follow plugin name == config name semantic in the whole codebase, but in most cases, I saw that they are the same.

Reply:
I didn't know we actually used `config.language` already. So thats a must to rename this feature. The name of the property should match the name of the plugin.